encrypted-media-respec.html

```diff
@@ -3611,7 +3612,7 @@
 
             <dt>User deletion of Key System storage</dt>
             <dd>
-              <p>User agents SHOULD present the user with a way to delete Key System storage for a specific <a def-id="origin"></a> or all origins.
+              <p>User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> or for all origins.
```

Comment:
My interpretation is "by origin" means for a specific origin.  So as written, this section says that the UA provides a feature for specific selected origins or across all origins.